py3status/modules/online_status.py

```diff
@@ -36,11 +36,27 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     format = '{state}'
```

Comment:
It'd have been nice if this was called `{icon}` not `{state}` I'd still argue to change it as it was only merged yesterday.